### PR TITLE
docs: added high-level project description

### PR DESCRIPTION
**Pull request by mstefferson**
# Context

Added high level project goals and get CI to pass

# Changes

* README updates
* Added package
* Add placeholder test (can remove later)

# Behaves Differently

* Passes CI

# Untested

* anything untested?

Closes #XXX.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,282 +16,42 @@ Clone the code to your machine using the standard Git clone command. If you have
...
-Important note: **Do not clone your code into Google Drive or DropBox**. There are known issues with MLFlow interacting with the file sync that is happening in the background. Clone to a directory that is not being synced by one of those services.
 
-## Docker
+## Project goal
 
-You will need to have Docker and docker-compose installed on your system to run the code.
+This project serves two primary goals:
 
-### Install Docker
+1. Make it easy to mock data for accelerate development for collaborative ML teams
```
Comment (mstefferson) [inline README.md]: to accelerate

Comment (tomNummy) [inline README.md]: what is the intended scale of this data? (order of magnitude)

File: README.md [hunk 2]
```diff
@@ -16,282 +16,42 @@ Clone the code to your machine using the standard Git clone command. If you have
...
+### 2. Generate data from causal structure
 
-### Retrieving Versions of Tracked Files
+Thinking of data from generative processes and having a playground to generate data has
+immense value in ML. Let's say you have a conceptual model (i.e. a causal DAG) of your
+system. What does data generated from this look like? Does this look like my data? What
+happens if I have a collider in my system? My linear model says this effect is positive,
+does that match the generative process? Is conditioning on other covariates leading me
+to draw the wrong conclusions? These are important questions, and ones we hope you're
+asking yourself..  Building simple models and testing them is the key to gaining
```
Comment (mstefferson) [inline README.md]: yourself

File: README.md [hunk 3]
```diff
@@ -16,282 +16,42 @@ Clone the code to your machine using the standard Git clone command. If you have
...
+intuition and to understand what's actually going on.
 
-1. `git checkout <branch>`
-2. `dvc pull` - sync local cache with remote cache based on current git HEAD
-3. `dvc checkout` - link files in local cache to appropriate location in project repo.
+This project aims to provide a nice user-face for engineers and scientists to mock data
+from a casual DAG:
```
Comment (jeffmm) [inline README.md]: causal

File: README.md [hunk 4]
```diff
@@ -16,282 +16,42 @@ Clone the code to your machine using the standard Git clone command. If you have
...
 
+DAG represented as a matrix -> Casual DAG -> generated data
```
Comment (jeffmm) [inline README.md]: Causal

**Review by jeffmm — APPROVED** [review]
LGTM

**Review by tomNummy — COMMENTED** [review]
One of the things I like to do when starting a project is to add a "why this is different from <insert name of popular open source project here>" section to help me stay focused on not re-inventing the wheel. What is the closest project to this one that we know of?

**Review by tomNummy — APPROVED** [review]
📖 📚 📘

**Pull request merged**